drivers/can/Kconfig.native_linux

```diff
@@ -0,0 +1,37 @@
+# Native Linux SocketCAN configuration options
+
+# Copyright (c) 2022 Martin Jäger <martin@libre.solar>
+# SPDX-License-Identifier: Apache-2.0
+
+config CAN_NATIVE_LINUX
+	bool "Native Linux SocketCAN Driver"
+	default y
+	depends on DT_HAS_ZEPHYR_NATIVE_LINUX_CAN_ENABLED
+	help
+	  Enable native Linux SocketCAN Driver
+
+if CAN_NATIVE_LINUX
+
+config CAN_NATIVE_LINUX_RX_THREAD_STACK_SIZE
+	int "Stack size for interrupt handler"
+	default 512
+	help
+	  Size of the stack used for internal thread which is ran for
+	  interrupt handling and incoming packets.
+
+config CAN_NATIVE_LINUX_RX_THREAD_PRIORITY
+	int "Priority for interrupt handler"
+	default 2
+	help
+	  Priority level of the internal thread which is ran for
```

Comment:
This description is a bit misleading: There is no interrupts involved here, but polling of the socket.